nxus-searcher-environment.js

```diff
@@ -0,0 +1,48 @@
+const NxusEnvironment = require('./nxus-environment')
+const tester = require('nxus-tester')
+
+class NxusSearcherEnvironment extends NxusEnvironment {
+  constructor(config) {
+    if (!config.testEnvironmentOptions) {
+      config.testEnvironmentOptions = {}
+    }
+    if (!config.testEnvironmentOptions.serverEnv) {
+      config.testEnvironmentOptions.serverEnv = {}
+    }
+    if (!config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__index) {
+      config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__index = 'searcher-jest-test'
+    }
+    if (!config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__host) {
+      config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__host = 'localhost:9200'
+    }
+    super(config)
+
+    this.searcherHost = config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__host
+    this.searcherIndexName = config.testEnvironmentOptions.serverEnv.nxus_storage__connections__searcher__index
+    this.searcherIndex = this.searcherHost+'/'+this.searcherIndexName
+  }
+
+  async setup() {
+    console.log("Setting up searcher index at", this.searcherIndex)
+    // But let nxus startup actually create the index with settings
+    await super.setup()
+    
+    this.global.tester.searcherRefresh = async () => {
```

Comment:
@ScottMaxson asks "why is this being added to the global here".

So this refresh function only makes sense to me at the intersection of tests/nxus/searcher, so I think it makes sense to live in this preset's environment. But I didn't think it should introduce a new global so I'm adding it into the `tester` namespace. Open to other suggestions. Maybe it makes sense as a method on searcher itself and I'm wrong about only using it in tests. In that case, tests would use `application.get('searcher').refreshIndex()` say.